src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidationsTest.java

```diff
@@ -258,6 +261,79 @@ void noCaseIdTest() {
         );
     }
 
+    @Test
+    void caseIdTest() {
+        checkValidation(
+            createCaseWith(b -> b.id(1L)),
+            true,
+            1L,
+            CallbackValidations::hasAnId,
+            null
+        );
+    }
+
+    @Test
+    void caseIdNoCaseDetailsTest() {
+        checkValidation(
+            null,
+            false,
+            1L,
+            CallbackValidations::hasAnId,
+            "Exception case has no Id"
+        );
+    }
+
+    @Test
+    void hasInvalidDataTest() {
+        CaseDetails caseDetails = createCaseWith(
+            b -> b
+                .id(null)
+                .caseTypeId("SERVICE_ExceptionRecord")
+                .jurisdiction("BULKSCAN")
+        );
+        Validation<Seq<String>, CaseDetails> res =
+            CallbackValidations.hasValidData(true, caseDetails);
+        assertThat(res.isValid()).isEqualTo(false);
```

Comment:
I think we should make it a test per error or verify that all possible errors are included in validation result.

Reply:
Added check for all validation results